### PR TITLE
Fixed leaking of passwords in ansible module invocation & logs

### PR DESCRIPTION
**Pull request by moonrail**
Hello folks,

just a small PR to fix an issue with current main module spec in module_utils/vcd.py, that leads to plaintext passwords in Ansible output.

Also added usage of "env_fallback" instead of "default" for Environment Variable fallbacks, to allow setting "required" to True on mandatory attributes like user, password, org & host.
<br>This way, users will now not be getting an error like "Login failed for user None to org None", but e.g.:
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "missing required arguments: user, password, org"}
```

By changing Exception-Handling from python exception to modules fail_json, any unsuccessfull login attempt will now be output in a clean manner, instead of a traceback.

### CODE REVIEW

File: module_utils/vcd.py [hunk 1]
```diff
@@ -26,6 +27,7 @@ def __init__(self, *args, **kwargs):
         argument_spec = vcd_argument_spec()
         argument_spec.update(kwargs.get('argument_spec', dict()))
         kwargs['argument_spec'] = argument_spec
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Comment (mukultaneja) [inline module_utils/vcd.py:30]: could you please call it in global space?

File: module_utils/vcd.py [hunk 2]
```diff
@@ -5,19 +5,20 @@
 from lxml import etree
 from pyvcloud.vcd.client import Client
 from pyvcloud.vcd.client import TaskStatus
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import AnsibleModule, env_fallback
 from pyvcloud.vcd.client import BasicLoginCredentials
 from ansible.module_utils.vcd_errors import VCDLoginError
```
Comment (mukultaneja) [inline module_utils/vcd.py:10]: please review unused imported modules.

File: module_utils/vcd.py (hunk 1 above)
Comment (moonrail) [reply module_utils/vcd.py:30]: Why global?
All modules inherit `VcdAnsibleModule` anyway, as it handles common module spec & login, so init will be called as soon as they're created.

File: module_utils/vcd.py (hunk 2 above)
Comment (moonrail) [reply module_utils/vcd.py:10]: Will do

File: module_utils/vcd.py (hunk 1 above)
Comment (mukultaneja) [reply module_utils/vcd.py:28]: correct but it is a module `urllib3` level setting and I think it would be better if we set it outside the class.

Comment (moonrail) [reply module_utils/vcd.py]: done, squashed commits

**Pull request merged**